DorisProject/src/app/pages/footer/footer.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss']
```

Comment:
Codacy has a fix for the issue: [Insert `,`](https://app.codacy.com/gh/SaraArenasV/DorisProjectFrontEnd/pullRequest?prid=9275970)
```suggestion
  styleUrls: ['./footer.component.scss'],
```